### PR TITLE
Add support for additional work in the direct container

### PR DESCRIPTION
**Pull request by isolgpus**
Noticed when running the server in direct mode, the additional work did not seem to be executed. This change just wires it up from the config.

### CODE REVIEW

File: src/test/java/com/aitusoftware/babl/integration/SingleWebSocketSessionDirectSessionContainerAcceptanceTest.java [hunk 1]
```diff
@@ -412,4 +409,32 @@ public void onConnectionClosed()
             latch.countDown();
         }
     }
+
+    private class AdditionalWorkTestHandler implements Agent
+    {
+        int additionalWorkCount = 0;
```
Comment (babl-ws) [inline src/test/java/com/aitusoftware/babl/integration/SingleWebSocketSessionDirectSessionContainerAcceptanceTest.java:415]: `private` and `volatile` please

File: src/test/java/com/aitusoftware/babl/integration/SingleWebSocketSessionDirectSessionContainerAcceptanceTest.java [hunk 2]
```diff
@@ -17,37 +17,19 @@
...
 import com.aitusoftware.babl.log.Logger;
 import com.aitusoftware.babl.user.ContentType;
 import com.aitusoftware.babl.user.EchoApplication;
-import com.aitusoftware.babl.websocket.Client;
-import com.aitusoftware.babl.websocket.ClientEventHandler;
-import com.aitusoftware.babl.websocket.ConnectionValidator;
-import com.aitusoftware.babl.websocket.ValidationResult;
-import com.aitusoftware.babl.websocket.ValidationResultPublisher;
-
+import com.aitusoftware.babl.websocket.*;
```
Comment (babl-ws) [inline src/test/java/com/aitusoftware/babl/integration/SingleWebSocketSessionDirectSessionContainerAcceptanceTest.java:23]: No wildcard imports, please revert back to single imports

File: src/main/java/com/aitusoftware/babl/websocket/SessionContainer.java [hunk 3]
```diff
@@ -144,18 +144,19 @@
 
     /**
      * Constructs a web-socket server that will dispatch messages to the supplied {@code Application}.
-     *
-     * @param application   the application that will process inbound messages
+     *  @param application   the application that will process inbound messages
```
Comment (babl-ws) [inline src/main/java/com/aitusoftware/babl/websocket/SessionContainer.java:147]: Remove leading space

File: src/main/java/com/aitusoftware/babl/websocket/SessionContainer.java [hunk 4]
```diff
@@ -144,18 +144,19 @@
...
      * @param sessionConfig configuration for web socket sessions
      * @param sessionContainerConfig  configuration for the web socket server
+     * @param additionalWork
```
Comment (babl-ws) [inline src/main/java/com/aitusoftware/babl/websocket/SessionContainer.java:150]: Add param comment please

**Review by babl-ws — CHANGES_REQUESTED** [review]
Thanks Jamie, just a couple of modifications needed.

**Comment by isolgpus** [comment]
Issues should be resolved I think. I'm surprised intellij was a bit aggressive with the import optimising

**Review by babl-ws — APPROVED** [review]

**Comment by babl-ws** [comment]
@isolgpus I've just enabled CI for PRs, can you make a no-op commit to your PR to check that it's working please?

**Pull request merged**